File: Serveur/controller/CSalon.py
from modele.Salon import Salon


def listeSalon(request):
    idUtil = request.form.get('idUtil')
    return Salon.listerSalonUtil(idUtil)

def creationSalon(request):

    nomSalon = request.form.get('nomSalon')
    idUtil = request.form.get('idUtil')

    return Salon.ajoutSalon(nomSalon, idUtil)


def rejoindreSalon(request):

    idUtil = request.form.get('idUtil')
    nomSalon = request.form.get('nomSalon')

    return Salon.rejoindreSalon(idUtil, nomSalon)


def listeMessageSalon(request):

    idSalon = request.form.get('idSalon')

    return Salon.listeMessageSalon(idSalon)

File: Site/controller/CSalon.php
<?php
    include("modele/Salon.php");

    function listerSalonUtilisateur($idUtil){
        return Salon::listerSalonUtilisateur($idUtil);
    }

    function creationSalon($nomSalon, $idUtil){
        return Salon::creationSalon($nomSalon, $idUtil);
    }
    function rejoindreSalon($idUtil, $nomSalon){
        return Salon::rejoindreSalon($idUtil, $nomSalon);
    }
?>